src/devices/machine/acorn_vidc.cpp

```diff
@@ -150,6 +159,7 @@ void acorn_vidc10_device::device_start()
 	save_item(NAME(m_sound_frequency_test_bit));
 	save_item(NAME(m_cursor_enable));
 	save_pointer(NAME(m_crtc_regs), CRTC_VCER+1);
+	save_pointer(NAME(m_crtc_raw_horz), 2);
 	m_data_vram = auto_alloc_array_clear(machine(), u8, m_data_vram_size);
```

Comment:
The `auto_alloc` functions are considered deprecated; `make_unique_clear` should be used here instead.

Reply:
Not touched by the PR, additionally I have no idea about how to pass a std::unique_ptr to a function in C++ (cfr. `draw()` ).

Reply:
Pass it either as a reference or as a bare pointer, passing in &my_unique_ptr[0] from the calling code.